pilot/pkg/model/context.go

```diff
@@ -552,6 +550,9 @@ type NodeMetadata struct {
 	// Namespace is the namespace in which the workload instance is running.
 	Namespace string `json:"NAMESPACE,omitempty"`
 
+	// WorkloadName specifies the name of the workload represented by this node.
+	WorkloadName string `json:"WORKLOAD_NAME,omitempty"`
```

Comment:
This is just moving from BootstrapMeta to NodeMetadata. This split is only intended to avoid accidentally taking dependencies on "bootstrap only" metadata in pilot (which we want to avoid). 

In the new model, we use workloadname in pilot so we need to move it.